vnext/Microsoft.ReactNative/Views/ViewViewManager.cpp

```diff
@@ -504,6 +504,13 @@ void ViewViewManager::TryUpdateView(
   if (isControl == useControl && hadOuterBorder == hasOuterBorder)
     return;
 
+  // This short-circuits if we are already a control and no longer need to be.
+  // Chances are if something was a control at one point in time, it will need
```

Comment:
Are you sure there aren't scenarios where we expect the ViewControl to be gone, and leaving it there will cause unexpected behavior?

Reply:
iirc this whole logic was added to deal with background / border / rounded corners etc. so pls make sure setting/clearing those still works : )

Reply:
Updated the test plan to include a test of the existing `view.tsx` sample. Behavior is identical.

Reply:
I also added another test plan to test the interaction between `focusable` and accessible. See the PR summary test plan for more details about "bugs". Hoping @chiaramooney can clarify whether either are actual bugs or expected behavior.

Reply:
Ah, actually nevermind. I realize that this is short sighted. The test plan only modifies one prop at a time. If border and control related props are set simultaneously, then this would break. I'll make a more targeted fix.